src/generator.rs

```diff
@@ -67,48 +75,81 @@ impl Generator {
                     .include_file(Self::TONIC_INCLUDE_FILE)
                     .compile(&protos, includes)?;
             }
+            Generator::Protoc { language, out_dir } => {
+                let mut protoc_cmd = std::process::Command::new(protoc);
+
+                match language {
+                    Language::Python => {
+                        protoc_cmd.arg("--python_out").arg(out_dir);
+                    }
+                }
+
+                // Setting proto path causes protoc to replace occurrences of this string appearing in the
+                // path of the generated path with that provided by output path
+                // e.g. if input proto path is proto/vendor/units/units.proto and the proto path is 'proto'
+                // and the --python_out is 'proto/build/gen' then the file will be output to
+                // proto/build/gen/vendor/units/units.py
+
+                // Add the proto_path arg now
+                protoc_cmd.arg("--proto_path").arg("proto/vendor"); // We need both of these if we want "vendor" to be removed, and it has to come first
+                protoc_cmd.arg("--proto_path").arg("proto");
+
+                // Add the proto files we want code generated for
+                protoc_cmd.args(&protos);
+
+                tracing::info!(":: running {protoc_cmd:?}");
```

Comment:
```suggestion
                tracing::info!(":: running protoc");
```

Reply:
Or you could even delete it entirely tbh? We are logging out that we compiled each dependency in `generate`

Reply:
(See other comment, moved to debug as it will be needed if we ever get protoc generating things in the wrong folder, as proto_path and out-directory if not set correctly can cause confusion)